viewing_party/party.py

```diff
@@ -0,0 +1,243 @@
+
+######################################################
+                # Wave 1 #
+######################################################
+
+#makes movie dictionary and returns it
+def create_movie(movie_title, genre, rating):
+    #initialize variables
+    movie_dict = {}
+
+    #check if movie title, genre and rating are string/float variables
+    #returns movie_dict or None
+    if type(movie_title) == str and type(genre) == str and type(rating) == float:
+        movie_dict['title'] = movie_title
+        movie_dict['genre'] = genre
+        movie_dict['rating'] = rating
+    else:
+        return None
+    return movie_dict
```

Comment:
Consider first testing for the edge case, and then moving the main functionality of your function outside the `if/else`. This programming pattern is called a guard clause and can among other things enhance the readability of your code.
 
```suggestion
    if not movie_title and not genre and not rating:
        return None
        
    movie_dict = {}
    movie_dict["title"] = movie_title
    movie_dict["genre"] = genre
    movie_dict["rating"] = rating
    return movie_dict
```